server/php/UploadHandler.php

```diff
@@ -154,7 +149,13 @@ public function __construct($options = null, $initialize = true, $error_messages
                     //'upload_url' => $this->get_full_url().'/thumb/',
                     // Uncomment the following to force the max
                     // dimensions and e.g. create square thumbnails:
-                    //'crop' => true,
+                    // 'auto_orient' => true,
+                    // 'crop' => true,
+                    // 'jpeg_quality' => 70,
+                    // 'no_cache' => true, (there's a caching option, but this remembers thumbnail sizes from a previous action!)
+                    // 'max_width' => 0, (this would mean width is adjusted automatically - keeping aspect ratio to a set max_height)
+                    // 'max_heigh' => 200,
```

Comment:
I would prefer not to add the max_width / max_height comment lines (max_heigh is also not a valid property -
 missing the "t"), as they would conflict with the existing properties below.
The other comment lines are good additions.